packages/components/src/components/bal-list/bal-list-item-accordion-head/bal-list-item-accordion-head.tsx

```diff
@@ -22,7 +22,12 @@ export class ListItemAccordionHead {
    */
   @Event() balAccordionChange!: EventEmitter<boolean>
 
+  private list?: HTMLBalListElement | null
+  private iconName!: string
+
   connectedCallback() {
+    this.list = this.el.closest('bal-list')
+    this.iconName = this.list?.classList.contains('is-in-main-nav') ? 'nav-go-down' : 'plus'
```

Comment:
Please use a Prop. Otherwise, we have no type-safety and no clear interface and are depending on the outside